evap/evaluation/models.py

```diff
@@ -362,12 +370,12 @@ def textanswer_set(self):
     @property
     def open_textanswer_set(self):
         """Pseudo relationship to all text answers for this course"""
-        return TextAnswer.objects.filter(contribution__in=self.contributions.all(), checked=False)
+        return TextAnswer.objects.filter(contribution__in=self.contributions.all(), state='')
```

Comment:
i think it is very weird that the default state is the empty string. is there a good reason for that?